### DISCUSSION TITLE
dnscrypt-proxy logs and permissions

### DISCUSSION THREAD

**Discussion by romz987** [DNSCrypt/dnscrypt-proxy#2234] [Q&A]
Hello!
Can you help me resolve problem with dnscrypt-proxy permissions?
I edited *.toml: uncommented logs and added blacklist file.
When dsncrypt-proxy starts (systemctl start dnscrypt-proxy) it creates /dnscrypt-proxy symlink and /private directory.

> lrwxrwxrwx   1 root   root  dnscrypt-proxy -> private/dnscrypt-proxy
> drwx------       3  root   root  private/

I can explore /private dir and logs just under root user (or after sudo su), but I can not cd /private dir under any other user with sudo.
How can I fix it?

And is it really important root:root owner for "blocked-names.txt" file for adding it when dnscrypt-proxy starts?

Thank you.

**Comment by jedisct1** [comment]
> When dsncrypt-proxy starts (systemctl start dnscrypt-proxy) it creates /dnscrypt-proxy symlink and /private directory.

Errr... what?

There's nothing in the `dnscrypt-proxy` code that does anything like it.

Where did you download it from? What operating system is it?

Permissions are not important as long as the server can read the configuration files, and write to the cache directory.

**Comment by romz987** [comment]
OS Arch Linux

> 6.0.6-arch1-1

Installed by pacman:

> Name            : dnscrypt-proxy
> Version         : 2.1.2-1
> Description     : A flexible DNS proxy, with support for encrypted DNS protocols
> Architecture    : x86_64
> URL             : https://github.com/DNSCrypt/dnscrypt-proxy

From Community Rep

**Comment by romz987** [comment]
You can download and explore package:
https://filetransfer.io/data-package/HZNfBAVx#link

I uploaded it from /var/cache/packman/pkg/

**Comment by jedisct1** [comment]
Since this is not part of this software, you'd better ask that package's maintainer, or on an Arch Linux forum.

**Comment by romz987** [comment]
Ok, I will try
